### ISSUE TITLE
TypeScript not working

### ISSUE THREAD

**Issue by guybedford** [jspm/jspm-cli#2452]
```
jspm install -g typescript
```

when run gives the following errors:

```
error TS2318: Cannot find global type 'Array'.

error TS2318: Cannot find global type 'Boolean'.

error TS2318: Cannot find global type 'Function'.

error TS2318: Cannot find global type 'IArguments'.

error TS2318: Cannot find global type 'Number'.

error TS2318: Cannot find global type 'Object'.

error TS2318: Cannot find global type 'RegExp'.

error TS2318: Cannot find global type 'String'.


Found 8 errors.
```

**Comment by guybedford** [comment]
@irustm sorry I misunderstood your comment previously, if you want to investigate this that would be amazing.

The first step would be to work out what TypeScript code path is trying to load the global types, and to see why that codepath isn't working in the jspm conversion.

This sounds like a typical conversion bug though, since it's likely a dynamic require scenario, where ideally we want to convert dynamic requires into dynamic imports for the conversion.

The hard part about that though is that it likely isn't an async function. This is also related to https://jspm.org/docs/integrations#one-weird-trick-to-support-jspm-in-tools.

The static analysis does support wildcards like `require('./asdf/' + unknown)` so this will likely be fixed by ensuring we can handle this wildcard case in the dew conversion statically as opposed tot rying to treat it as a dynamic import (https://github.com/jspm/babel-plugin-transform-cjs-dew).

**Comment by irustm** [comment]
omg, 
```bash
> jspm install -g typescript

err  TypeError: Cannot read property 'warn' of undefined
    at Project.checkGlobalBin (/Users/irustm/.nvm/versions/node/v11.14.0/lib/node_modules/jspm/lib/project.js:138:22)
```

**Comment by guybedford** [comment]
Woah, thanks. Fixed in https://github.com/jspm/jspm-cli/commit/dbf8fde36e7eaa030a7b52e5a47e937c8b9b3a9b.

**Comment by guybedford** [comment]
I also went ahead and fixed the support for TypeScript in https://github.com/jspm/jspm-cli/commit/cdf7144f9fd38f0469ad9b59d6c88086c749d591. So `jspm install -g typescript && tsc` now works!

**Comment by guybedford** [comment]
Released in 2.0.0-beta.4.
